sdks/python/build.gradle

```diff
@@ -89,6 +89,7 @@ task depSnapshot {
 
 task dependencyUpdates {
   dependsOn ':dependencyUpdates'
+  dependsOn buildPython
```

Comment:
I think this tasks does not install all dependencies we need. 

We'd like to install all extras (there are currently five of them: docs, test, gcp, interactive, aws. You can see them in `setup.py`